### ISSUE TITLE
[pre-alpha] Group Scheduler Analytics page Throws Exception

### ISSUE THREAD

**Issue by NuclearTacos** [SparkDevNetwork/Rock#4010]
### Description
Group Scheduler Analytics page throws an exception in Beta 10.1 and Pre-Alpha 11.0

```
Specified TimeUnitType is invalid for this SlidingDateRangePicker control.
at Rock.Web.UI.Controls.SlidingDateRangePicker.set_TimeUnit(TimeUnitType value) in C:\Source\Repos\Rock\Rock\Web\UI\Controls\Pickers\SlidingDateRangePicker.cs:line 632
   at Rock.Web.UI.Controls.SlidingDateRangePicker.set_DelimitedValues(String value) in C:\Source\Repos\Rock\Rock\Web\UI\Controls\Pickers\SlidingDateRangePicker.cs:line 753
   at RockWeb.Blocks.GroupScheduling.GroupSchedulerAnalytics.LoadFilterFromUserPreferencesOrURL() in c:\inetpub\wwwroot\Blocks\GroupScheduling\GroupSchedulerAnalytics.ascx.cs:line 368
   at RockWeb.Blocks.GroupScheduling.GroupSchedulerAnalytics.OnLoad(EventArgs e) in c:\inetpub\wwwroot\Blocks\GroupScheduling\GroupSchedulerAnalytics.ascx.cs:line 204
   at System.Web.UI.Control.LoadRecursive()
   at System.Web.UI.Control.LoadRecursive()
   at System.Web.UI.Control.LoadRecursive()
   at System.Web.UI.Control.LoadRecursive()
   at System.Web.UI.Control.LoadRecursive()
   at System.Web.UI.Control.LoadRecursive()
   at System.Web.UI.Control.LoadRecursive()
   at System.Web.UI.Control.LoadRecursive()
   at System.Web.UI.Control.LoadRecursive()
   at System.Web.UI.Control.LoadRecursive()
   at System.Web.UI.Page.ProcessRequestMain(Boolean includeStagesBeforeAsyncPoint, Boolean includeStagesAfterAsyncPoint)
```

### Steps to Reproduce
1. Go to Group Scheduler Analytics page.

**Expected behavior:**

No Exception

**Actual behavior:**

Exception

### Versions
* **Rock Version:** Beta 10.1, Pre-Alpha 11.0
* **Client Culture Setting:** en-us

**Comment by jennifertroeger** [comment]
I received this same error in my 10.1 beta testing.

**Comment by nairdo** [comment]
@NuclearTacos First of all thank you for finding this issue.  (We really don't like it when these sorts of bugs get past us.)

We're going to re-patch v10.1 (something we don't normally like to do since it's already in beta), but for those of you who may already have 10.1 running in production let me know (DM me) and I'll get you the updated Rock.DLL (and a block that fixes #4017) with the fix (https://github.com/sparkdevnetwork/rock/commit/29bb1913c3ed1480576760540d4bada620a10bd2) in it.
